### PR TITLE
Skip download if file already exists

### PR DESCRIPTION
**Pull request by NicksonYap**
Useful if network connection lost, or disk space full

### CODE REVIEW

File: src/duke/downloadDukeMTMC.m [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 options = weboptions('Timeout', 60);
 
+FILE_ALREADY_EXISTS_ERROR = 'Skipped - File exists !!! (delete to redownload)\n'; % Where to store DukeMTMC (160 GB)
```
Comment (ergysr) [inline src/duke/downloadDukeMTMC.m]: This is not necessarily an error, rather a warning.

File: src/duke/downloadDukeMTMC.m [hunk 2]
```diff
@@ -41,7 +43,12 @@
     for k = 1:length(urls)
         filename = sprintf('%sground_truth/%s',dataset.savePath,filenames{k});
         fprintf([filename '\n']);
-        websave(filename,urls{k},options);
+        url = urls{k};
```
Comment (ergysr) [inline src/duke/downloadDukeMTMC.m]: This block repeats in many places. Would be useful to make it a function, e.g. downloadFile(filename, urls{k}, options)

File: src/duke/downloadDukeMTMC.m (hunk 1 above)
Comment (NicksonYap) [reply src/duke/downloadDukeMTMC.m]: Nice catch there :sweat_smile:

File: src/duke/downloadDukeMTMC.m [hunk 3]
```diff
@@ -41,7 +43,12 @@
...
+        if ~isfile(filename)
```
Comment (ergysr) [inline src/duke/downloadDukeMTMC.m]: This should additionally check if the md5 hash of the existing file matches the one on the server. http://vision.cs.duke.edu/DukeMTMC/data/videos/md5sum.txt

**Comment by NicksonYap** [comment]
@ergysr  force pushed

able to merge using the forced commit?

**Pull request merged**